modules/rgbd/src/pose_graph.cpp

```diff
@@ -4,44 +4,70 @@
 
 #include "pose_graph.hpp"
 
+#include <fstream>
 #include <iostream>
```

Comment:
`iostream` should not be used in OpenCV modules.

> `std::cout`

Use `CV_LOG_*` macro instead.